src/EmptyValidator.js

```diff
@@ -1,14 +1,19 @@
+const _ = require('lodash');
 const Validator = require('./Validator');
 
 class EmptyValidator extends Validator {
 
     constructor() {
+        if (_.size(arguments)) {
+            throw new Error('There must be no arguments specified.');
+        }
+
         super();
-        this.withErrorMessage('Must not be empty');
+        this.withErrorMessage('Value must be empty.');
     }
 
     isValid(value){
-        return value === undefined || value === null || value === '' || value === 0;
+        return value === undefined || value === null || value === '' || value === 0 || (_.isArray(value) && value.length === 0) || (_.isObject(value) && _.values(value).length === 0);
```

Comment:
Can all these be replaced by `_.isEmpty()`?

Reply:
Not possible. `_.isEmpty() ` considers all numbers as empty.

Reply:
Should `yarn.lock` be added to `.gitignore` then?

Reply:
Done both.